lib/high_voltage/page_collector.rb

```diff
@@ -0,0 +1,47 @@
+module HighVoltage
+
+  require 'find'
+
+  class PageCollector
+
+    attr_reader :page_ids, :content_path
+
+    def initialize(content_path)
+      @content_path = content_path.to_s
+    end
+
+    def page_ids
+      page_ids = []
+
+      Find.find(content_path) do |f|
```

Comment:
@bentoncreation good call. I think your right that it's thinking of enumerable over `Find`